docs/src/contribute/contribute-core-rule.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Contribute to Core Rules
+eleventyNavigation:
+    key: contribute core rule
+    parent: contribute to eslint
+    title: Contribute to Core Rules
+    order: 10
+---
+
+The ESLint core rules are the rules included in the ESLint package.
+
+## Rule Writing Documentation
+
+For full reference information on writing rules, refer to [Custom Rules](../extend/custom-rules). Both custom rules and core rules have the same structure and API. The primary difference between core and custom rules are:
+
+1. Core rules are included in the `eslint` package.
+1. Core rules must adhere to the structure documented on this page.
+
+## Core Rule Structure
```

Comment:
```suggestion
## Rule Writing Documentation

For full reference information on writing rules, refer to [Custom Rules](../extend/custom-rules). Both custom rules and core rules have the same API. The primary difference between core and custom rules are:

1. Core rules are included in the `eslint` package.
1. Core rules must adhere to the conventions documented on this page.

## File Structure
```

This could be confusing because first we say that core rules _have the same structure as custom rules_, but then that core rules _must adhere to the structure documented on this page_.